<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecognitionResultPane</class>
 <widget class="QWidget" name="RecognitionResultPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="controlsViewLayout">
       <item>
        <layout class="QVBoxLayout" name="objectsViewLayout">
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>300</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Objects</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="FullResultLayout">
         <item>
          <widget class="QLabel" name="FullResultLabel">
           <property name="text">
            <string>Full result</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FullResultLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="PossibleCharsLayout">
         <item>
          <widget class="QLabel" name="PossibleCharsLabel">
           <property name="text">
            <string>Selection possible chars</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PossibleCharsLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="BarChartLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
